AGPs/Association-2020-Budget.md

```diff
@@ -0,0 +1,131 @@
+---
+AGP: N/A
+Title: Association 2020 Budget
+Author: burrrata (@burrrata)
+Status: Stage III
+Track: Finance
+Created: 2019-11-19
+---
+
+# Association 2020 Budget
+
+## Address of the transfer recipient
+
+https://mainnet.aragon.org/#/association
+
+## Amount of the transfer
+
+The association would like to request:
+- 1,500,000 DAI
+- 375,000 ANT
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+1
+
+## Purpose of the transfer
+
+The Aragon Association is the legal entity that oversees the Aragon Project's development during the decentralization process.
+
+The Association's responsibilities include: 
+- Spreading the values in the [Aragon Manifesto](https://blog.aragon.org/the-aragon-manifesto-4a21212eac03/).
+- Creating value for ANT holders.
+- Managing the [project's treasury](https://transparency.aragon.org).
+- Ensuring the long-term sustainability of the Aragon project.
+- Ensuring Aragon's adoption by overseeing development and awareness efforts.
+- Nurturing Aragon's governance and curating [governance proposals](https://aragon.org/project/governance).
+- Decentralizing all roles and responsibilities in order to transfer full governance power to ANT holders.
+
+In order to fulfill the above responsibilities for the Aragon Network, the Association needs capital to support its team and cover Network related expenses. These expenses include legal costs, HR, communications, accounting, banking, auditing, market making and more. While the goal is to decentralize these responsibilities to independent DAOs (as much as possible), current they are essential to the health and producitivity of the Aragon Network.
+
+Therefore, it requests authorization to use the following funds from the Aragon treasury:
+
+- 1,500,000 DAI for operational costs.
```

Comment:
This nearly doubles the operational costs compared to last year. Would be good to understand the implied changes in scope that justify this growth as well as how and when this will start trending in the other direction. https://github.com/aragon/AGPs/blob/master/AGPs/AGP-11.md